### ISSUE TITLE
Multiple directives on same page share event(s)

### ISSUE THREAD

**Issue by umdstu** [begriffs/angular-paginate-anything#89]
There doesn't appear to be a way to differentiate between paginated elements on the same page when responding to their emitted events, ie. pagination:loadPage. I suggest adding an additional required scoped field, identifier, that is used in setting up events to listen on. See this branch: https://github.com/umdstu/angular-paginate-anything/tree/new-identifier-attribute

**Comment by begriffs** [comment]
Good idea. Love that you documented the changes too. Can you open a PR for this?

Also it would be good if the identifier was optional for when there is only one pagination element on the page.

**Comment by umdstu** [comment]
I'll update the change this week.

**Comment by bacbos** [comment]
@umdstu Seems like a useful feature, do you still have plans of sending in a PR?
